docassemble/ALToolbox/PhoneNumberDataType.py

```diff
@@ -28,7 +28,7 @@ class PhoneNumber( CustomDataType ):
 $.validator.addMethod( 'phone', validatePhoneNumber );
 """
     jq_rule = 'phone'
-    jq_message = 'You need to enter a valid phone number.'
+    jq_message = 'Invalid input. Did you enter a "+" before your number?'
```

Comment:
This message is a bit of a problem if it confuses people from the US. If they add a '+' to the front of a correct number, their country becomes egypt or something like that.

I can see this happening when a person types in an invalid number, gets that message, incorrectly adds the plus. They then get the message again because their number is still invalid, so they fix other parts of it and then hit 'submit' without removing the '+'.

Users do crazy things, but that's who we're building it for. That said, we may need to leave this detail for future people to work on.